### PR TITLE
List appending

### PR DESCRIPTION
**Pull request by briduffy**
The list titles and descriptions all append, but, my tests more or less failed -- definitely mostly failed, but I tried to implement what I think you wanted.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -14,7 +14,8 @@ gem 'bootsnap', '>= 1.1.0', require: false
 gem 'slim'
 
 group :development, :test do
-  gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
+  gem 'sqlite3', '1.3.13'
```
Comment (reecebisel) [inline Gemfile:17]: The app is already using postgres, so we shouldn't need to add this. If it was added so the tests would run then we just needed to set up the test db.

File: app/models/list.rb [hunk 2]
```diff
@@ -1,3 +1,3 @@
 class List < ApplicationRecord
-  has_many :lists
+  has_many :lists_items, dependent: :destroy
```
Comment (reecebisel) [inline app/models/list.rb:2]: Good catch. Orphaned list items would really ruin some stuff.

File: test/controllers/lists_controller_test.rb [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  end
+
+  test "should get new" do
+    get new_list_url
+    assert_response :success
+  end
+
+  test "should create list" do
+    assert_difference('List.count') do
+      post lists_url, params: { list: { title: @list.title, description: @list.description } }
```
Comment (reecebisel) [inline test/controllers/lists_controller_test.rb:17]: these look really good. Nice to see that you where able to get up to speed on minitest

**Review by reecebisel — COMMENTED** [review]
Most of this looks good. Just the one thing about the sqlite gem

**Review by reecebisel — CHANGES_REQUESTED** [review]
remove the sqlite gem and this should be good to go.

File: Gemfile (hunk 1 above)
Comment (briduffy) [reply Gemfile:17]: This was pretty dumb of me, huh?

**Pull request closed without merging**